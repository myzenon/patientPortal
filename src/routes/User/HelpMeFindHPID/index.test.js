import React from 'react'
import { HelpMeFindHPID } from './index'
import ReactGA from 'react-ga';

   // This should be part of your setup
   ReactGA.initialize('UA-121783543-1', {
       'debug': true
     });;

jest.mock("react-ga")

const propsFromRedux = {
    page: {
        data: {
            isCaregiver: true
        }

    },
}
test('renders correctly', () => {
    const props = { ...propsFromRedux }
    const wrapper = render(
        <HelpMeFindHPID {...props} />
    )
    expect(toJson(wrapper)).toMatchSnapshot()
})