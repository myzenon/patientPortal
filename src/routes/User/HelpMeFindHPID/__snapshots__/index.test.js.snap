// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  class="container-fluid"
>
  <div
    class="content container"
  >
    <p>
      You should have a Health Plan Card
    </p>
    <p>
      that looks like this one:
    </p>
    <p>
      <img
        alt="Health Plan Card"
        src="health-plan-card-help.png"
      />
    </p>
    <p>
      <a
        href="/caregiver-patient-lookup"
      >
        <button
          class="button btn btn-secondary"
        >
          FOUND IT!
        </button>
      </a>
    </p>
    <p>
      <button
        class="button btn btn-secondary"
        type="button"
      >
        DIDN'T FIND IT
      </button>
    </p>
  </div>
</div>
`;
